<!DOCTYPE html>
<html lang="en">
    <head>
        <meta charset="UTF-8" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
        <title>Ibreeding</title>
        <link rel="stylesheet" href="../../../assets/css/main.css" />
        <link rel="stylesheet" href="../../../assets/css/theme.css" />
        <link rel="stylesheet" href="../../../components/shared/calculator.css" />
        <link rel="stylesheet" href="../../../components/shared/plot.css" />
        <link rel="stylesheet" href="../../../components/shared/slider.css" />
        <script src="https://cdn.plot.ly/plotly-latest.min.js"></script>
        <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;600&display=swap" rel="stylesheet">
        <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.7/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-LN+7fdVzj6u52u30Kp6M/trliBMCMKTyK833zpbD+pXdCLuTusPj697FH4R/5mcr" crossorigin="anonymous">
        <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.7/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>
    </head>
    <script src="../../../popdynWebsite/js/controller.js"></script>
    <body>
        <!-- Boostrapnav bar -->
        <nav class="navbar navbar-expand-lg bg-body-tertiary">
        <div class="container-fluid">
            <a class="navbar-brand" href="#">PopGen</a>
            <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
            <span class="navbar-toggler-icon"></span>
            </button>
            <div class="collapse navbar-collapse" id="navbarNav">
            <ul class="navbar-nav">
                <li class="nav-item">
                <a class="nav-link active" aria-current="page" href="#">Home</a>
                </li>
                <li class="nav-item">
                <a class="nav-link" href="#">Features</a>
                </li>
                <li class="nav-item">
                <a class="nav-link" href="#">Pricing</a>
                </li>
                <li class="nav-item">
                <a class="nav-link disabled" aria-disabled="true">Disabled</a>
                </li>
            </ul>
            </div>
        </div>
        </nav>

        <main class="d-flex">
        <!-- Sidebar que rola com o conteúdo -->
        <nav class="flex-shrink-0 p-3 text-bg-dark" style="width: 250px;">
            <h4 class="text-white">Content Table</h4>
            <hr>
            <ul class="nav nav-pills flex-column">
            <li class="nav-item"><a class="nav-link text-white" href="#gene-genotype">1. Models of Population Growth</a></li>
            <li class="nav-item"><a class="nav-link text-white" href="#hardy-weinberg">2. Randomly Mating Population</a></li>
            <li class="nav-item"><a class="nav-link text-white" href="#x-linked">3. Inbreeding</a></li>
            <li class="nav-item"><a class="nav-link text-white" href="#two-loci">4. Correlation Between Relatives and Assortative Mating</a></li>
            <li class="nav-item"><a class="nav-link text-white" href="#wahlund">5. Selection</a></li>
            <li class="nav-item"><a class="nav-link text-white" href="#finite-pop">6. Population in Approximate Equilibrium</a></li>
            <li class="nav-item"><a class="nav-link text-white" href="#finite-pop">7. Properties of a Finite Population</a></li>
            <li class="nav-item"><a class="nav-link text-white" href="#finite-pop">8. Stochastic Processes in the Change of Gene Frequencies</a></li>
            <li class="nav-item"><a class="nav-link text-white" href="#finite-pop">9. Distribution of gene Frequencies in Population</a></li>
            </ul>
        </nav>
            <nav class="flex-grow-1 p-4">
            <!-- Conteúdo Principal -->
            <section id="Inbreeding">
                <h1>Models of Population Growth</h1>

            </section>
            <section id="DecreaseHet">
                <h2>3. DecreaseHet</h2>
                <p>Inbreeding occurs when mates are more closely related than they would be if they had been chosen at random from the population. Related individuals have one or more ancestors in common, so the extent of inbreeding is related to the amount of ancestry that is shared by the parents of the inbred individuals. Alternatively stated, the degree of inbreeding of an individual is determined by the proportion of genes that his parents have in common. 
                    An immediate consequence of this sharing of parental genes is that the inbred individual will frequently inherit the same gene from each parent <strong>(identical by descent)</strong>. Thus inbreeding increases the amount of homozygosity. So one observable effect of inbreeding is that recessive genes, previously hidden by heterozygosity with dominant alleles, will be expressed. Since most such genes are harmful in one way or another, inbreeding usually leads to a decrease in size, fertility, vigor, yield, and fitness <strong>(inbreeding depression)</strong>. There are also likely to be loci segregating in a population where a heterozygote is fitter than either of the two corresponding homozygotes. In this case, too, inbreeding leads to decreased fitness. 
                    Another consequence of consanguineous mating within the population is greater genetic variability, since genes tend to be concentrated in the same individuals. Usually, because of the correlation between genotype and phenotype, this leads to an increase in phenotypic variability. 
                    Inbreeding may follow either of two patterns. There may be a certain amount of consanguineous mating within a population, with the consequences just mentioned. On the other hand, the inbreeding may be such as to break the population into subgroups. An extreme example is continued self-fertilization in which the population (if it is constant size and each parent contributes equally to the next generation) is divided into a set of subpopulations of one individual each. Likewise, a pattern of repeated sib mating could lead to a series of isolated populations of size 2. As a third example, there may be a natural population which is divided into isolated subpopulations, within each of which mating is random or nearly so. The effect will be that each subpopulation becomes more homozygous, and therefore the whole population does. The individual subpopulations become more uniform genetically; but, since they become homozygous for different genes, the population as a whole becomes more variable. Of course, there may be only partial isolation, with intermediate consequences. 
                    A point that at first seems paradoxical is that within a subpopulation there is an increase in homozygosity despite the fact that mating within this group is random. The reason is that there are random changes in the frequencies of the individual alleles and these, on average, lead to a decrease in heterozygosity. As an extreme example, self-fertilization can be regarded as random mating (i.e., random combination of gametes) within a population of one. The gene frequencies at different, previously heterozygous loci change from \(\frac{1}{2}\) to 0 or 1. 
                    Whether inbreeding leads to subdivision or not, it can be measured in the same way - by Wright's (1922) coefficient of inbreeding, \(f\), which measures the proportion by which the heterozygosity has been decreased or the probability of two alleles at a locus to be identical by descent. As we shall show later, other population properties can also be related to \(f\). However, before discussing \(f\), we shall illustrate with two simple examples the effect of continued inbreeding.</p>

                <h2>3.1 Decrease in Heterozygosity with Inbreeding</h2>

                <p>The qualitative effect of continued inbreeding can be seen by examining the most extreme form, self-fertilization. In a self-fertilization population, the progeny of homozygotes are like their parents, whereas the progeny of heterozygotes are \(\frac{1}{2}\) heterozygotes and \(\frac{1}{4}\) each of the two homozygous types. 
                    Thus, in each generation the proportion of the heterozygous loci is reduced by half and the homozygous types are correspondingly increased.</p>

                \[
                H_t = \frac{H_0}{2^t}
                \]

                <p>And the homozygosity increases as:</p>

                \[
                \begin{align}
                \text{Proportion AA} &= D + \frac{H}{2} \\
                \text{Proportion aa} &= R + \frac{H}{2}
                \end{align}
                \]

                <p>The key point here is that while the genotype frequencies change, allele frequencies remain constant. Inbreeding restructures the gene pool, but does not shift it. 
                    This is illustrated:</p>

                \[
                \begin{array}{c c c c} \hline
                \text{Generation} & AA & Aa & aa \\ \hline
                0 & D & H & R \\
                1 & D+\frac{H}{4} & \frac{H}{2} & R+\frac{H}{4} \\
                2 & D+\frac{3H}{8} & \frac{H}{4} & R+\frac{3H}{8} \\
                3 & D+\frac{7H}{16} & \frac{H}{8} & R+\frac{7H}{16} \\
                4 & D+\frac{15H}{32} & \frac{H}{16} & R+\frac{15H}{32} \\
                \text{Limit} & D+\frac{H}{2} & 0 & R+\frac{H}{2} \\ \hline
                \end{array}
                \]

                <p>If \(H_0\) is the initial proportion of heterozygotes, the proportion after \(t\) generations of self-fertilization is \(\frac{H_0}{2^t}\). If the original population were panmictic, with \(AA, Aa\) and \(aa\) genotypes in the proportions \(p^2, 2pq\) and \(q^2\) \((p + q = 1)\), the individual line eventually becomes homozygous. The probability of being \(AA\) is \(D + \frac{H}{2} = p^2 + pq = p(p+q) = p\); likewise, the probability of being \(aa\) is \(q\). Thus, the population becomes broken into separate lines, each homozygous for one of the other of the genes in the ratio of their original frequencies in the population. Notice one other fact: There has been no change in the gene frequency. Inbreeding per se does not change the proportions of the various genes, only the way they are combined into homozygous and heterozygous genotypes. 
                    With less extreme forms of inbreeding, the results are similar, though the change in heterozygosity is less rapid. The results for continued brother-sister mating are shown below. Again there is a decrease in the proportion of heterozygotes, with the amount deducted being divided equally and added to the two homozygous types. 
                    These results may be obtained by writing out all the possible mating generation after generation, as was done by the early investigators (Fish 1914 and Jennings 1916). This and several other systems of recurrent inbreeding were worked out by these authors. The papers are now mainly of historical interest since more general methods are available.</p>

                \[
                \begin{array}{c|c|c|c|c} \hline
                \text{Generation} & \text{Relative Heterozygosity} & \text{Decrease in Heterozygosity} & \text{Rates of Change In Heterozygosity} & \\
                t & \frac{H_t}{H_0}=P & \frac{H_0-H_t}{H_0}=f & \frac{H_t}{H_{t-1}} & \frac{H_{t-1}-H_t}{H_{t-1}} \\ \hline
                0 & 1 & 0 & 1 & 0 \\
                1 & \frac{2}{2} & 0 & 1 & 0 \\
                2 & \frac{3}{4} & \frac{1}{4} & \frac{3}{4}=0.750 & 0.250 \\
                3 & \frac{5}{8} & \frac{3}{8} & \frac{5}{6}=0.833 & 0.167 \\
                4 & \frac{8}{16} & \frac{8}{16} & \frac{8}{10}=0.800 & 0.200 \\
                5 & \frac{13}{32} & \frac{19}{32} & \frac{13}{16}=0.812 & 0.188 \\
                6 & \frac{21}{64} & \frac{43}{64} & \frac{21}{26}=0.808 & 0.192 \\
                \text{Limit} & 0 & 1 & \lambda=0.809 & 0.191 \\ \hline
                \end{array}
                \]

                <p>In this example, the heterozygosity follows a simple rule. The numerator in successive generations is given by the Fibonacci series in which each term is the sum of the two preceding terms, while the denominator doubles each generation. The reduction in heterozygosity, expressed as a fraction of the initial heterozygosity, is the same regardless of the initial gene frequencies and, as shall show later, the number of alleles. 
                    The relative heterozygosity, \(\frac{H_t}{H_0}\) has been called by Wright (1951) the <em>panmictic index</em> for which he used the letter \(P\). \(1 - P\) is the inbreeding coefficient, for which Wright has used the letter \(F\). We shall use the lower case \(f\) in order to reserve \(F\) for multiple-locus inbreeding effects. 
                    The last two columns give the rate of change in heterozygosity. Notice that the ratio \(\frac{H_t}{H_{t-1}}\) after a few oscillations rapidly approaches a constant value. The limiting value of the ratio of heterozygosity to that in the previous generation is usually designated by \(\lambda\). 
                    In summary, the conceptual synthesis can be seen as follows:</p>

                \[
                \begin{array}{|c|c|c|} \hline
                \text{Feature} & \text{Selfing} & \text{Sib mating} \\ \hline
                \text{Heterozygosity decline} & \text{Halves each generation} & \text{Declines ~20\%/generation} \\ \hline
                \text{Genotype frequency change} & \text{fast} & \text{slower} \\ \hline
                \text{Allele frequency change} & \text{None} & \text{None} \\ \hline
                \text{Population structure effect} & \text{Becomes a set of fully homozygous lines} & \text{Subgroups drift toward homozygosity} \\ \hline
                \end{array}
                \]
                    <table id="heterozygosityTable">
                    <thead>
                        <tr>
                            <th>Generation</th>
                            <th>Relative Heterozygosity (H_t / H_0)</th>
                            <th>Decrease in Heterozygosity (f)</th>
                            <th>Rate of Change (H_t / H_{t-1})</th>
                            <th>Delta Rate (1 - H_t / H_{t-1})</th>
                        </tr>
                    </thead>
                    <tbody>
                        <!-- Data will be inserted here -->
                    </tbody>
                </table>

                <div id="plotRelativeHeterozygosity" class="plot-container" style="width:100%;max-width:700px;margin:auto;"></div>
                <div id="plotDecreaseHeterozygosity" class="plot-container" style="width:100%;max-width:700px;margin:auto;"></div>
                <div id="plotRateOfChange" class="plot-container" style="width:100%;max-width:700px;margin:auto;"></div>
                <div id="plotDeltaRate" class="plot-container" style="width:100%;max-width:700px;margin:auto;"></div>
            </section>

            <section id="WrightInbreeding">
                <h2>3.2 Wright's Inbreeding Coefficient, \(f\)</h2>

                <p>Wright's (1922) original derivation of the inbreeding coefficient, \(f\), was through correlation analysis. An alternative approach using only probability rules has been developed by Haldane and Moshinsky (1939), Cotterman (1940), and Malécot (1948). They distinguish between two ways in which an individual can be homozygous for a given locus. The two homologous genes may be:</p>
                <ol>
                    <li>Alike in state, that is to say, indistinguishable by any effect they produce (or perhaps, when molecular genetics has become sufficiently precise, alike in their nucleotide sequence).</li>
                    <li>Identical by descent, in that both are derived from the same gene in a common ancestor.</li>
                </ol>

                <p>We follow the notation of Cotterman in designating an individual whose two homologous genes are identical by descent as <em>autozygous</em>. If the two alleles are of independent origin (as far as known from our pedigree information), the individual is <em>allozygous</em>. The effect of inbreeding is to increase that part of the homozygous without being autozygous, if the two homologous genes are alike in state but not identical by descent. Conversely, an autozygous individual can be heterozygous for this locus if one of the two alleles has mutated since their common origin, although this is negligibly rare if only a small number of generations is being considered. 
                            The inbreeding coefficient, \(f\), is defined as the probability that the individual is autozygous for the locus in question. Alternatively stated, it is the probability that a pair of alleles in the two gametes that unite to form the individual are identical by descent. 
                            An individual with inbreeding coefficient \(f\) has a probability \(f\) that the two genes at a particular locus are identical and a probability \(1-f\) that they are not identical, and therefore independent. If they are independent, the frequencies of the genotypes will be given by the binomial formula. If they are identical, the frequencies of the gene pairs will be simply the frequencies of the alleles in the population. Thus, for two alleles, \(A_1\) and \(A_2\), with frequencies \(p_1\) and \(p_2 \: (p_1 + p_2 = 1)\), the genotype frequencies are:</p>

                \[
                \begin{array}{cc} \hline
                & \text{Allozygous} & \text{Autozygous} \\ \hline
                \text{Homogygous, } A_1A_1: & p_1^2(1-f) & p_1 f \\
                \text{Homogygous, } A_1A_2: & 2p_1p_2(1-f) &   \\
                \text{Homogygous, } A_2A_2: & p_2^2(1-f) & p_2f   \\
                \text{Total} & 1-f & f   \\ \hline
                \end{array}
                \]

                <p>Notice that when \(f = 0\) these formulas reduce to the usual Hardy-Weinberg proportion. When \(f=1\) the population is completely homozygous. Thus \(f\) ranges from 0 in a randomly mating population to 1 with complete homozygosity. How to compute \(f\) from a pedigree will be shown later. 
                    Multiple alleles introduce no difficulty. The genotype frequencies are a natural extension of the results for two alleles. The frequencies are:</p>

                \[
                A_iA_i: p_i^2 (1-f) + p_i f
                \]

                <p>For homozygous genotypes, and:</p>

                \[
                A_iA_j: 2p_ip_j (1-f)
                \]

                <p>For heterozygous genotypes. The expected proportion of heterozygous genotypes with inbreeding coefficient \(f\), \(H_f\), is given by:</p>

                \[
                H_f = \sum_{i \ne j} p_i p_j (1-f) = H_0 (1-f)
                \]

                <p>Given that:</p>

                \[
                f = \frac{H_0 - H_f}{H_0}
                \]

                <p>Where \(H_0\) is a constant equal to the proportion of heterozygotes expected with random mating \((f=0)\). The summation is over all combinations of values of \(i\) and \(j\) except when these are equal. 
                    This proves the assertion made earlier that the inbreeding coefficient measures the fraction by which the heterozygosity has been reduced. We have written the formula as if, when \(f=0\), the population is in Hardy-Weinberg proportions. However, for any measured \(f\) (as determined, for example, from a pedigree), the heterozygosity, \(H\), is \(H_0(1-f)\), where \(H_0\) is whatever the heterozygosity would have been in the absence of the observed inbreeding. To be concrete, the inbreeding coefficient for the child of a cousin marriage is \(\frac{1}{16}\) (as we shall show later); therefore the child of such a marriage is \(\frac{15}{16}\) as heterozygous as if his parents had the same relationship as a random pair in this population. 
                    There is a simple relationship between the correlation coefficient, \(r\), and the inbreeding coefficient, \(f\). If we assign numerical values to each allele, then the inbreeding coefficient, \(f\), is the correlation between these values in a pair of uniting gametes. In fact, Wright's original derivation of the inbreeding coefficient was through correlation methods. 
                    The relationship between \(r\) and \(f\) can be shown in the following way. For convenience, we assign the value 1 to \(A_1\) and 0 to allele \(A_2\), though we would get the same result with any values. The calculations are shown below. 
                    Since the sum of the genotype frequencies is equal to 1, the weighted sum and the mean of any are the same. For example, the sum (and mean) of egg value, \(X\), is \([p_2^2(1-f)+p_2f](0)+[p_1p_2(1-f)](1)+ [p_1p_2(1-f)](0)+[p_1^2(1-f)+p_1f](1)\), which after some algebraic simplification reduces to \(p_1\). The other calculations are given in the table, using the standard formula for calculation of \(r\).</p>

                \[
                \begin{array}{c c c c c c c c} \hline
                \text{Egg} & \text{Sperm} & \text{Frequency} & X & Y & X^2 & Y^2 & XY\\  \hline
                A_1 & A_2 & p_2^2(1-f)+p_2f & 0 & 0 & 0 & 0 & 0 \\
                A_1 & A_2 & p_1p_2(1-f)& 1 & 0 & 1 & 0 & 0\\
                A_2 & A_1 & p_2p_1(1-f) & 0 & 1 & 0 & 1 & 0\\
                A_1 & A_1 & p_1^2(1-f)+p_1f & 1 & 1 & 1 & 1 & 1\\  \hline
                \text{Sum or mean} & 1 & p_1 & p_1 & p_1 & p_1 & p_1 & p_1^2(1-f)+p_1f\\  \hline
                \end{array}
                \]

                \[
                \bar X = p_1 p_2(1 - f) + p_1^2 (1 - f) + p_1 f = p_1
                \]

                <p>Likewise, \(\hat Y = \text{mean}(X^2) = \text{mean}(Y^2) = p_1\)</p>

                \[
                \begin{align}
                r_{xy} &= \frac{\bar{XY}-\bar X \bar Y}{\sqrt{(\text{mean}(X^2)- \bar X^2)(\text{mean}(Y^2)- \bar Y^2)}} \\
                r_{xy} &= \frac{p_1^2(1-f)+p_1f-p_1^2}{p_1 - p_1^2} = f
                \end{align}
                \]

                <p>Demonstration without restriction as to number of alleles and letting the contribution of the alleles differ. Furthermore, if the genic values are summed over \(k\) loci the covariance will be:</p>

                \[
                f \sum_k \sum_i p_{ik} a_{ik}^2
                \]

                <p>Where \(p_{ik}\) and \(a_{ik}\) are the frequency and value of the \(i\)th allele at the \(k\)th locus. This is \(f\) times the variance. Hence \(f\) is the expected value of the correlation between the genetic values of two uniting gametes, regardless of the number of loci and number of alleles under consideration. 
                    The equivalence of \(r\) and \(f\) suggests an interpretation of the correlation coefficient. If a measurement can be thought of as being the sum of a number of elements, then the correlation coefficient is the measure of the fraction of these elements that are common to the two measurements, the other elements being chosen at random. This interpretation is useful in many branches of science. In quantitative genetics, the elements can obviously be interpreted as cumulative acting genes.</p>

                \[
                V_x = \sum_i p_i a_i^2
                \]

                <p>Since the variance of the egg value is the sum of the squares of the allele values, each weighted by its frequency. \(V_y\) is the same.</p>

                \[
                COV_{XY} = (1-f) \left[  \sum_i p_i^2 a_i^2 + \sum_{i \ne j }p_i p_j a_i a_j\right] + f \sum_i p_i a_i^2
                \]

                <p>But the quantity in brackets is equal to \(\left[\sum_i p_i a_i \right]^2\) which is equal to 0, because the sum of the deviations from the mean is 0. Therefore,</p>

                \[
                COV_{XY} = f \sum_i p_i a_i^2
                \]

                <p>The correlation coefficient, being the ratio of the covariance to the geometric mean of the two variances (which in this case are the same), is \(f\), as was to be shown:</p>

                \[
                r_{XY} = \frac{COV_{XY}}{V_x} = f
                \]

            <h1>Genotype Frequencies and Heterozygosity</h1>
            <div id="plotGenotypeFrequencies" class="plot-container" style="width:100%;max-width:700px;margin:auto;"></div>
            <div id="plotHeterozygosity" class="plot-container" style="width:100%;max-width:700px;margin:auto;"></div>

            <h1>Equivalence of Inbreeding Coefficient (f) and Correlation Coefficient (r)</h1>

            <div id="plotCorrelation" class="plot-container" style="width:100%;max-width:700px;margin:auto;"></div>
    
            </section>
            <section id="OverlappingDiscrete">
            </section>
            <section id="OverlapppingContinous">
            </section>
            <section id="FisherMeasure">
            </section>
            <section id="PopRegulation">
            <section id="UnregulatedGrowth">
            </section>
            <section id="LogisticRegulation">
            </section>
            <section id="WeakControl">
            </section>
            <section id="SpaceFoodUtilization">
            </section>
            <section id="WeakerPopulationControl">
            </section>
            <section id="RegulationByEfficiency">
            </section>

            <!-- pagination -->
            <nav aria-label="...">
                <ul class="pagination justify-content-center">
                    <li class="page-item"><a href="#" class="page-link">Previous</a></li>
                    <li class="page-item active">
                    <a class="page-link" href="../CrowKimura/Popgen2_RandPop.html" aria-current="page">2</a>
                    </li>
                    <li class="page-item">
                        <a class="page-link" href="../CrowKimura/Popgen3_Inbreeding.html">3</a>
                    </li>
                    <li class="page-item"><a class="page-link" href="#">4</a></li>
                    <li class="page-item">
                        <a href="../CrowKimura/Popgen2_RandPop.html" class="page-link">Next</a>
                    </li>
                </ul>
                </nav>
                <!-- bootstrap footer -->
                <div class="container">
                    <footer class="d-flex flex-wrap justify-content-between align-items-center py-3 my-4 border-top">
                        <p class="col-md-4 mb-0 text-body-secondary">© 2025 Company, Inc</p>
                        <a href="/" class="col-md-4 d-flex align-items-center justify-content-center mb-3 mb-md-0 me-md-auto link-body-emphasis text-decoration-none" aria-label="Bootstrap">
                            <svg class="bi me-2" width="40" height="32" aria-hidden="true"><use xlink:href="#bootstrap"></use></svg>
                        </a>
                        <ul class="nav col-md-4 justify-content-end">
                            <li class="nav-item"><a href="#" class="nav-link px-2 text-body-secondary">Home</a>
                        </li>
                            <li class="nav-item"><a href="#" class="nav-link px-2 text-body-secondary">Features</a>
                            </li>
                            <li class="nav-item">
                                <a href="#" class="nav-link px-2 text-body-secondary">Pricing</a>
                            </li>
                            <li class="nav-item">
                                <a href="#" class="nav-link px-2 text-body-secondary">FAQs</a>
                            </li>
                            <li class="nav-item">
                                <a href="#" class="nav-link px-2 text-body-secondary">About</a>
                            </li>
                        </ul>
                    </footer>
                </div>
            </nav>
        </main>

            <script src="../simulation/simulation3.js"></script>
            <script src="../../../js/utils/plotly_theme.js"></script>
            <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.7/dist/js/bootstrap.bundle.min.js" integrity="sha384-ndDqU0Gzau9qJ1lfW4pNLlhNTkCfHzAVBReH9diLvGRem5+R9g2FzA8ZGN954O5Q" crossorigin="anonymous"></script>

    </body>
</html>
